.github/workflows/dialyzer-cache.yml

```
@@ -0,0 +1,73 @@
+name: Dialyzer with Caching
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)